<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JFCTA9I" failures="2" tests="3" name="Error Validations Test" time="19.938" errors="0" timestamp="26 Mar 2023 08:23:40 GMT">
  <testcase classname="bb.tests.ErrorValidationsTest" name="IntentdedFailureForScreenshotFeatureValidation" time="2.214">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at bb.tests.ErrorValidationsTest.IntentdedFailureForScreenshotFeatureValidation(ErrorValidationsTest.java:43)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- IntentdedFailureForScreenshotFeatureValidation -->
  <testcase classname="bb.tests.ErrorValidationsTest" name="IntentdedFailureForScreenshotFeatureValidation" time="2.101">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at bb.tests.ErrorValidationsTest.IntentdedFailureForScreenshotFeatureValidation(ErrorValidationsTest.java:43)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- IntentdedFailureForScreenshotFeatureValidation -->
  <testcase classname="bb.tests.ErrorValidationsTest" name="LoginErrorValidation" time="0.837"/>
</testsuite> <!-- Error Validations Test -->
